core/stopWatch.go

```diff
@@ -91,14 +91,14 @@ func (sw *StopWatch) GetMeasurementsMap() map[string]float64 {
 	return output
 }
 
-// GetMeasurement returns the measurement (duration in seconds) by identifier
-func (sw *StopWatch) GetMeasurement(identifier string) float64 {
+// GetMeasurement returns the measurement by identifier
+func (sw *StopWatch) GetMeasurement(identifier string) time.Duration {
```

Comment:
Is it ok(expected) that this function will only return elapsed time from when you called `Stop`?

Reply:
Indeed, the stop watch was designed that way. `GetMeasurement` returns delta between `Start` and `Stop`, disregarding the moment you call it.